kubernetes-ingress/templates/controller-daemonset.yaml

```diff
@@ -176,7 +176,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           {{- if .Values.controller.extraEnvs -}}
-            {{- toYaml .Values.controller.extraEnvs | nindent 12 }}
+          {{- toYaml .Values.controller.extraEnvs | nindent 10 }}
```

Comment:
@dkorunic these have to be 10 (not 12) since the list of env vars is already established above it and that list is not indented after the `env:` key (i.e. the list looks like the following):

```
env:
- one
- two
```

Instead of the "correct" formatting:

```
env:
  - one
  - two
```